uni/lib/controller/parsers/parser_library_occupation.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:convert';
+
+import 'package:http/http.dart';
+import 'package:uni/model/entities/library_occupation.dart';
+
+Future<LibraryOccupation> parseLibraryOccupationFromSheets(
+    Response response) async {
+  final json = response.body.split('\n')[1]; // ignore first line
+  const toSkip =
+      'google.visualization.Query.setResponse('; // this content should be ignored
+  const numFloors = 6;
+  final LibraryOccupation occupation = LibraryOccupation(0, 0);
+
+  final jsonDecoded =
+      jsonDecode(json.substring(toSkip.length, json.length - 2));
+
+  for (int i = 0; i < numFloors; i++) {
+    int floor, max;
+    try {
+      floor = jsonDecoded["table"]["rows"][i]["c"][0]["v"].toInt();
+      max = jsonDecoded["table"]["rows"][i]["c"][1]["v"].toInt();
+      occupation.addFloor(FloorOccupation(i + 1, floor, max));
+    } catch (e) {
+      occupation.addFloor(FloorOccupation(i + 1, 0, 0));
```

Comment:
I think it is because of this... It tries to parse some null value on the table value and it fails giving it occupation and capacity 0.

Reply:
@jlcrodrigues isn't the max value available on the spreadsheet always?